src/lib/dvf/bindApi.js

```diff
@@ -31,6 +31,7 @@ module.exports = () => {
     createOrderMessage: compose(require('../stark/createOrderMessage')),
     sign: compose(require('../stark/starkSign')),
     signAuth: compose(require('../stark/starkSignAuth')),
+    singAmmFundingOrder: require('../stark/singAmmFundingOrder')(dvf),
```

Comment:
same typo `sing` instead of `sign`

Reply:
fixed in bba3f9f49865cfe56a72b8f516231b5433754c08